app/presenters/licences/view-licence-bills.presenter.js

```diff
@@ -0,0 +1,45 @@
+'use strict'
+
+/**
+ * Formats data for the `/licences/{id}/bills` view licence bill page
+ * @module ViewLicenceBillsPresenter
+ */
+
+const { formatLongDate } = require('../base.presenter')
+
+/**
+ * Formats data for the `/licences/{id}/bills` view licence bill page
+ *
+ * @returns {Object} The data formatted for the view template
+ */
+function go (bills) {
+  return {
+    activeTab: 'bills',
+    bills: _formatBillsToTableRow(bills)
+  }
+}
+
+function _formatCurrencyToGBP (amount) {
+  return amount.toLocaleString('en-gb', {
```

Comment:
This is not doing to two decimal places. 

Need to see how other areas of the code are handling this